Query/ProxiSpectrumReader.cs

```diff
@@ -128,7 +128,7 @@ public List<ProxiSpectrum> Retrieve()
                         //injection time
                         if (ionInjectionTime != null)
                         {
-                            proxiSpectrum.AddAttribute(accession: "MS:1000927", name: "ion injection time",
```

Comment:
@xjasg Can you update your branch? Some of the terms on your side are outdated; we fixed them in our previous release.